docs/badge.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+
+    <link rel="stylesheet" href="/styles/main-style.css">
+    <link rel="stylesheet"
+      href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/default.min.css">
+</head>
+<body>
+    <nav class="navigation-bar">
+        <div class="navigation-logo"><img src="/Images/main logo.png"></div>
+        <div class="navigation-text">
+            <div class="text-one">Docs</div>
+            <div class="text-two">Support</div>
+        </div>
+    </nav>
+
+    <aside class="sidebar">
+        <h2>Components</h2>
+        <ul>
+           <li><a href="avatar.html">Avatar</a></li> 
+            <li><a href="alert.html">Alert</a></li>
+            <li><a href="badge.html">Badge</a></li>
+            <li><a href="button.html">Button</a></li>
+            <li><a href="card.html">Card</a></li>
+            <li><a href="image.html">Image</a></li>
+            <li><a href="input.html">Input</a></li>
+            <li><a href="text-utilities.html">Text Utilities</a></li>
+            <li><a href="lists.html">Lists</a></li>
+            <li><a href="grid.html">Grid</a></li>
+    
+        </ul>   
+    </aside>
+
+    <div class="main main-container">
+        <div class="description">
+            <p1>Badges are used to highlight an item's status for quick recognition.Badges are usually seen in social media apps to indicate the status of whether a person is online or offline. They are also used to indicate number of unread notifications. 
+            </p1>
+        </div>
+
+
+        <div class="badge-container-size">
+            <p>Badge on icon</p>    
+            <div class="badge-icon">
+                <div><img src="/Images/envelop icon.png"><p2 class="badge badge-icon-one">4</p2></div>
```

Comment:
In general p1 & p2 tags doesn't exist in HTML. So ,here you are just creating custom elements. Some browsers might support them and some might not. Instead, you can use p tag and apply styling using different classes. [Check here](https://teamtreehouse.com/community/why-does-h1-make-the-letters-bigger-but-p1-doesnt-define-a-paragraph)